packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -44,6 +46,19 @@ const Redirect = ({
                     pathname: routes.onboarding,
                     search: url_query_string,
                 });
+                RudderStack.track(
```

Comment:
are we sure that this url params called once?

Reply:
This one is specific to the use case when user arrive from email verification link to the app, so it should only call once and as far as I checked, it was only calling once, do you know any case where it could happen twice in our app ?

Reply:
you are right, it should happen once. just wanted to make sure that we are not sending this event more than once

Reply:
Confirmed. This only happens once